libraries/designsystem/src/main/kotlin/io/element/android/libraries/designsystem/theme/components/Button.kt

```diff
@@ -118,6 +119,14 @@ fun TextButton(
     leadingIcon = leadingIcon
 )
 
+@Composable
+fun InvisibleButton(
```

Comment:
I think it's a bit weird to have an `InvisibleButton` api?

Reply:
I can't think of a better idea, let's keep this for now.